docs/ref/contrib/gis/functions.txt

```diff
@@ -261,13 +261,16 @@ right-hand rule.
 ``GeoHash``
 ===========
 
-.. class:: GeoHash(expression, **extra)
+.. class:: GeoHash(expression, precision=None, **extra)
 
 *Availability*: PostGIS, SpatiaLite (LWGEOM)
 
 Accepts a single geographic field or expression and returns a `GeoHash`__
 representation of the geometry.
 
+The number of returned characters is controlled by the ``precision`` keyword
```

Comment:
Is "The ``precision`` keyword argument controls the number of decimal digits in the result." correct? ("digits" rather than "characters")

Is it worth mentioning the behavior in the default (None) case?